src/compiler/emitter.ts

```diff
@@ -9,9 +9,9 @@ namespace ts {
 
     // Flags enum to track count of temp variables and a few dedicated names
     const enum TempFlags {
-        Auto      = 0x00000000,  // No preferred name
+        Auto =      0x00000000,  // No preferred name

```

Comment:
I thought I was reverting this based on Ryan's comments, I'll check what it looked like before to be sure